lib/cc/engine/analyzers/violation.rb

```diff
@@ -8,8 +8,8 @@ module Analyzers
       class Violation
         attr_reader :issue
 
-        def initialize(base_points, issue, hashes)
-          @base_points = base_points
+        def initialize(language, issue, hashes)
+          @language = language
```

Comment:
Then what do you think of using the name `analyzer` anywhere that `language` or `language_strategy` is currently used. That might clear up the confusion.